src/module/item/weapon/data.ts

```diff
@@ -148,11 +162,10 @@ interface WeaponSystemData
     expend: number | null;
     reload: {
         value: WeaponReloadTime | null;
-        /** Whether the ammunition (or the weapon itself, if thrown) should be consumed upon firing */
-        consume: boolean | null;
         /** A display label for use in any view */
         label: string | null;
     };
+    ammunition: WeaponAmmunitionData | null;
```

Comment:
Let's go with `ammo` where possible to save on some ink

Reply:
Sure. Ammo was my preference, I only called it ammunition cause it seemed more official.